version/version.go

```diff
@@ -58,7 +56,7 @@ func Get() Info {
 		GoVersion:       runtime.Version(),
 		Compiler:        runtime.Compiler,
 		Platform:        fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
-		AzureSdkVersion: azuresdkversion.Number,
+		AzureSdkVersion: "SDK track2",
```

Comment:
There is no single version string provided any longer for the SDK (except arguably in an `internal` package that's not accessible). So I think all we can do is hard-code it from now on.